internal/volume/utils.go

```diff
@@ -0,0 +1,26 @@
+package volume
```

Comment:
Needs the copyright boilerplate in the new files.

Reply:
Added, thanks.